src/wrapper/SerializationWrapper.cpp

```diff
@@ -67,6 +67,70 @@ char fs_serializer_openmode(void* serializer)
     return ' ';
 }
 
+int fs_serializer_metainfo(void* serializer)
```

Comment:
The name suggest is returning a metainfo object, but it seems that is returning a size, 
how about ```fs_serializer_metainfo_size```